FLEDGE.md

```diff
@@ -121,6 +122,23 @@ The `adComponents` field contains the various ad components (or "products") that
 The browser will provide protection against microtargeting, by only rendering an ad if the same rendering URL is being shown to a sufficiently large number of people (e.g. at least 100 people would have seen the ad, if it were allowed to show).  As discussed in the [Outcome-Based TURTLEDOVE](https://github.com/WICG/turtledove/blob/master/OUTCOME_BASED.md) proposal, this threshold applies only to the rendered creative; all of the metadata associated with ad bidding is not under any such restriction.  Since a single interest group can carry multiple possible ads that it might show, the group will have an opportunity to re-bid another one of its ads to act as a "fallback ad" any time its most-preferred choice is below threshold.  This means that a small, specialized interest group that is still below the daily-update threshold could still choose to participate in auctions, bidding with a more-generic ad until the group becomes large enough.
 
 
+#### 1.3 Permission Delegation
+
+When one domain calls joinAdInterestGroup() or leaveAdInterestGroup() for an interest group with a different owner, the browser will fetch the URL https://owner.domain/.well-known/interest-group/permissions/domain.of.frame.that.called.the.method, without credentials, using the Network Partition Key of the frame that invoked the method. The fetched response should have a JSON MIME type and be of the format:
```

Comment:
Building the domain into the path strikes me as odd.  Are .well-known URLs not supposed to have query parameters?  I'd've guessed something like `https://owner.domain/.well-known/interest-group/permissions?domain=domain.of.frame.that.called.the.method`.

Also there has been some discussion indicating that there might be interest in the future of this sort of delegation being allowed for some IGs and not others.   Seems like using a URL with query parameters seems better for future permission refinement.

Reply:
I've switch to a query param.  I don't view them as logically being any different than a path, and don't recall seeing .well-known requests with query params, but I don't think there's any requirement, and I could well just not be aware of cases that do that.

I'd be concerned about sending too many requests if we include all IGs in the query param.  Regardless, that's an issue for another day (with this format, could even allow per-group rules in the dictionary).

Thanks for the feedback!